pkg/volume/volume.go

```diff
@@ -162,6 +162,9 @@ type BlockVolumeMapper interface {
 	// at attacher.Attach() and attacher.WaitForAttach().
 	// This may be called more than once, so implementations must be idempotent.
 	SetUpDevice() (string, error)
+
+	// Map maps the block device path for the specified spec and pod.
```

Comment:
Do you need a symmetric `UnmapVolume` call too?

Reply:
Yeah, i thought about that too.   But there is an `unmapper.Teardown()` method defined where custom code could go.

Reply:
Do we need to do any similar moving around of common logic?